src/main/java/com/singlestore/jdbc/client/ColumnDecoder.java

```diff
@@ -36,7 +36,7 @@ static ColumnDecoder decodeStd(ReadableByteBuf buf) {
 
     buf.skip(); // skip length always 0x0c
     short charset = buf.readShort();
```

Comment:
Looks like it always returns binary charset id = 63 no matter what @@character_set_server is set. I checked for 8.5.16. I think we need reconsider of usage this parameter for blob display size calculation. As an option to read @@character_set_server variable on connection establishment